department-of-veterans-affairs/content-build
# VACMS-13222: Display new 'Where' format for events

## Description
For this ticket we are adding a new address format along with google maps link to direction. To accomplish this, we have added an update to the graphQL query to add the fieldAddress to the fieldFacilityLocation to pull the address into the event data obj. And we made the necessary changes on the FE drupal templates for events.

## Testing done & Screenshots

Testing URLs:
/outreach-and-events/events/60333/
/outreach-and-events/events/60524/

Before:
![Outreach_And_Events___Women_s_Resource_Fair_With_Veterans_Justice_Outreach_Program_-_Los_Angeles__CA___Veterans_Affairs_before](https://github.com/department-of-veterans-affairs/content-build/assets/42885441/bc094c38-6078-4c0d-9cb8-1489604aebb4)


After:
![Outreach_And_Events___Women_s_Resource_Fair_With_Veterans_Justice_Outreach_Program_-_Los_Angeles__CA___Veterans_Affairs](https://github.com/department-of-veterans-affairs/content-build/assets/42885441/f6e45585-453f-4be4-8317-2cd923d1ad42)


## QA steps

What needs to be checked to prove this works?  
Verify that event pages have new address format and link to google maps. 
Verify that VA and non VA facility events both show the new formatted address. 

What needs to be checked to prove it didn't break any related things?  
Verify that event pages are still working properly.


## Acceptance criteria
- [x] Address is presented as described above, including Directions link 
- [x] Optional building/floor/room text is displayed if provided by CMS
- [x] Google maps directions link is present / working for all addresses
- [x] VA facilities additionally display the name of the VA facility
  - [x] If the VA facility has a web page (eg VAMC), the facility name is hyperlinked to that page (see Eng Notes above)
  - [x] Otherwise, there is no hyperlink (eg VC Outstation)
- [ ] Design review required
- [ ] A11y review required
